curriculum/challenges/english/14-responsive-web-design-22/learn-responsive-web-design-by-building-a-piano/612ebcba99bfa46a15370b11.md

```diff
@@ -7,7 +7,19 @@ dashedName: step-27
 
 # --description--
 
-Now you need to make it responsive. Add a `@media` query with a `max-width` of `768px`.
+The `@media` at-rule, also known as a media query, is used to conditionally apply CSS. Media queries are commonly used to apply CSS based on the viewport width using the `max-width` and `min-width` properties.
+
+In the below example the padding is applied to the `.card` class when the viewport is `960px` wide and below.
+
+```css
+@media (max-width: 960px) {
+  .card {
+    padding: 2rem;
+  }
+}
+```
+
+Add a `@media` query with a `max-width` of `768px`.
```

Comment:
```suggestion
Add a media query that will be applied when the viewport is `768px` wide and below.
```

Not required, but I wonder if we should change this to match the verbiage of the example so the camper has to think a bit about what they are actually writing.

Reply:
I wouldn't be opposed to that. The hint has most of the syntax and the example should help as well.